tests/files/odl/feed.json

```diff
@@ -81,6 +84,12 @@
               "copy": false,
               "print": false,
               "tts": false
+            },
+            "availability": {
+              "state": "unavailable",
+              "until": "2000-05-04T03:02:01Z",
+              "detail": "a detailed reason",
+              "reason": "http://terms.example.org/unavailable"
```

Comment:
Minor: Maybe use one of the reason codes from here... https://registry.opds.io/reason

Similar in other test feeds.